Formula/manga-g.rb

```diff
@@ -0,0 +1,15 @@
+class MangaG < Formula
+  desc "check some mangos"
+  homepage "https://manga-g.pages.dev"
+  url "https://github.com/manga-g/manga-g/archive/refs/tags/v0.1.0alpha.tar.gz"
```

Comment:
We don't ship alpha software. https://docs.brew.sh/Acceptable-Formulae#stable-versions